hw/ip/otbn/data/bignum-insns.yml

```diff
@@ -661,6 +684,11 @@
     The memory address must be aligned to WLEN bits.
     Any address that is unaligned or is above the top of memory results in an error (setting bit `bad_data_addr` in `ERR_BITS`).
     Any `*grs2` value greater than 31 before executing the instruction results in an error (setting bit `illegal_insn` in `ERR_BITS`) and no store or optional increment occurring.
+  errs:
+    - A `CALL_STACK` error from using `x1` as `grs1` or `grs2` when the call stack is empty.
```

Comment:
That's equal to the "grs12-call-stack" anchor (if you want).

Reply:
Ahah, but that's defined in a different file :-) I tried that at first and took a few seconds to work out why the YAML parsing was failing!